v3/src/components/graph/adornments/univariate-measures/univariate-measure-adornment-model.ts

```diff
@@ -27,6 +27,7 @@ export const UnivariateMeasureAdornmentModel = AdornmentModel
   .props({
     measures: types.map(MeasureInstance),
     showMeasureLabels: false,
+    showTipOnOverlapHover: false,
```

Comment:
I'm now wondering if adding this property to the models is at least flirting with overkill? The component could check whether the adornment is of type `Mean` or `Median` instead of using this property.

Reply:
The primary reason for storing something in an MST model is if it's user-modifiable state that should be serialized with the document. If it's not user-modifiable state, then storing it with the document just bloats the document unnecessarily. In this case it seems like a static property of the adornment determinable from the type, in which case it could be handled with a utility function that checked the type of the adornment or by adding an optional property to the adornment registration.

Reply:
@kswenson That makes a lot of sense, thanks 👍

I'm not sure how much this changes anything, but I only just realized that in my zeal to simplify things, I accidentally lost an important part of the feature -- the part that checks if the other adornment's line is within 2px of the current adornment's line. Sorry for the premature review request. I've switched this to a draft PR and temporarily removed @bfinzer from the Reviewers list. I'll work on getting that missing piece back.

Reply:
@kswenson @bfinzer Sorry again for the mix up. I've updated the PR with a new commit that drops the model changes entirely and adds a function to `UnivariateMeasureAdornmentHelper` for determining if an adornment is blocking another from receiving mouseover events.